addons/sys_intercom/stringtable.xml

```diff
@@ -151,6 +151,11 @@
             <French>Téléphone d'infanterie reçu %1</French>
             <Turkish>Piyade Telefonu Alındı %1</Turkish>
         </Key>
+        <Key ID="STR_ACRE_sys_intercom_infantryPhoneUnavailable">
+            <English>Infantry Phone used by someone else</English>
```

Comment:
```suggestion
            <English>Infantry Phone already in use</English>
```

Reply:
German one already had the same semantic meaning and italian would only become longer by rephrasing.